r/R/expression.R

```diff
@@ -29,8 +29,17 @@
   # stringr spellings of those
   "str_length" = "utf8_length",
   "str_to_lower" = "utf8_lower",
-  "str_to_upper" = "utf8_upper"
+  "str_to_upper" = "utf8_upper",
   # str_trim is defined in dplyr.R
```

Comment:
Can you update this comment to say dplyr-functions.R?

Reply:
Done